FlightJournal.Web/Controllers/TrainingLogController.cs

```diff
@@ -13,7 +13,7 @@ namespace FlightJournal.Web.Controllers
 {
     public class TrainingLogController : Controller
     {
-        private FlightContext db = new FlightContext();
+        private readonly FlightContext db = new FlightContext();
```

Comment:
en note jeg mener ikke readonly vil blive respekteret hvis det var for at forsøge at skrive beskytte

Reply:
Nej, det er jeg med på - det var bare en reaktion på en Resharper (?) advisory om at feltet kunne være readonly.